doc/api/os.md

```diff
@@ -451,6 +451,8 @@ system. This differs from the result of `os.homedir()`, which queries several
 environment variables for the home directory before falling back to the
 operating system response.
 
+Throws a [`SystemError`](https://nodejs.org/api/errors.html#errors_system_errors) if user has no `username` or `homedir`.
```

Comment:
1.  For internal links, we usually use relative URLs (so that they properly work inside versioned docs) and bottom references. Also, long lines need to be wrapped at 80 characters maximum. So this line may have this format:

```md
Throws a [`SystemError`][] if a user has no `username` or `homedir`.
```

And you can add this line in the list at the bottom of the file (before the `[`process.arch`]: process.html#process_process_arch` line):

```md
[`SystemError`]: errors.html#errors_system_errors
```

Reply:
Thanks! Updated to use relative link.